### PR TITLE
Fix some major bugs in xhc-whb04b-6 user component

### PR DESCRIPTION
**Pull request by jasenk2**
1.There is wrong code for LEAD position of feed rotary switch.
2.There was major mistake in parse procedure Pendant::processEvent values are tested against button codes for invalid code but object is send as null to actual Pendant::processEvent so segfault coredump was produced

### CODE REVIEW

**Comment by alkabal** [comment]
Well i really need to check what is true for lead but for the rest i think you are right already.

Can you add this patch line1425 file pendant.cc ?
replace : 
            axisJogStepSize = feedButton.stepSize() * 0.001 * mHal.getFeedOverrideMaxVel();
with : 
            axisJogStepSize = feedButton.stepSize() * 0.0001 * mHal.getFeedOverrideMaxVel();

Can you check something at same time ?

The args B from cmd line does not work fine with associated var mIsStepMode_5_10

Thanks

**Comment by jasenk2** [comment]
ok Alkabal i will try

File: .gitignore [hunk 1]
```diff
@@ -38,3 +38,4 @@ oprofile*
 position.txt
 *.9
 *.glade.h
+/build/
```
Comment (jepler) [inline .gitignore]: The .gitignore file commited to git reflects only files that are created during the build process.  This path /build/ is not.  You can add your own ignores in $GIT_DIR/info/exclude -- see man gitignore(5) for more information.

File: configs/.gitignore [hunk 2]
```diff
@@ -13,3 +13,5 @@ position*.txt
 *.ini.expanded
 hallib
 *_cmds.hal
+/Rack_Tool_Changer_simplified/
+/MPG 1 DZIŁAJACY/
```
Comment (jepler) [inline configs/.gitignore]: see above comment.

**Pull request merged**